src/App.tsx

```diff
@@ -1,9 +1,41 @@
+import { Outlet } from 'react-router-dom';
+import styled from 'styled-components';
+import Menu from './components/Menu';
+
 function App() {
   return (
-    <div>
-      React Messanger
-    </div>
+    <Container>
+      <LeftBox>
+        <Menu />
+      </LeftBox>
+      <RightBox>
+        <Outlet />
+      </RightBox>
+    </Container>
   );
 }
 
 export default App;
+
+const Container = styled.div`
+  width: 26rem;
+  height: 46rem;
+
+  border-radius: 2.3rem;
+  box-shadow: 0px 0px 5px #444;
+
+  display: flex;
+`;
+
+const LeftBox = styled.div`
+  flex-basis: 20%;
+  background-color: #e3e0e0a8;
+  border-top-left-radius: 2.3rem;
+  border-bottom-left-radius: 2.3rem;
```

Comment:
이부분은
```suggestion
  border-radius: 2.3rem 0 0 2.3rem;
```
으로 한줄에 작성할 수 있을것같아요~!